### ISSUE TITLE
CHIT duplicated in operator memory?

### ISSUE THREAD

**Issue by baznikin** [Altinity/clickhouse-operator#1262]
I played around with operator and ClickHouseInstallationTemplates. Changed one template few times but can't get it to re-apply to already deployed ClickHouseInstallation. I even re-create CHI but still - no new changes was introduced from CHIT to CHI. At some moment I checked operator logs and notice it report 5 CHITs instead of 2:

```
I1027 13:17:19.829044       1 normalizer.go:137] applyCHITemplates():clickhouse/clickhouse:Found auto-templates num: 4
I1027 13:17:19.829060       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 13:17:19.829078       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 13:17:19.829099       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 13:17:19.829115       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 13:17:19.829129       1 normalizer.go:149] applyCHITemplates():clickhouse/clickhouse:Found manual-templates num: 1
I1027 13:17:19.829146       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 13:17:19.829176       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 13:17:19.829193       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 13:17:19.829210       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 13:17:19.829227       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: /clickhouse-secure-svc-lb. Selector: map[] matches labels: map[]
I1027 13:17:19.829250       1 normalizer.go:202] applyCHITemplates():clickhouse/clickhouse:Used templates count: 5
```

after operator restart it reports 2 CHITS:

```
I1027 13:19:13.928767       1 worker-chit-reconciler.go:38] addChit():clickhouse/clickhouse-secure-ssl
I1027 13:19:13.928795       1 worker-chit-reconciler.go:41] addChit():clickhouse/clickhouse-secure-ssl:Add CHIT: clickhouse/clickhouse-secure-ssl
I1027 13:19:13.928838       1 worker-chit-reconciler.go:38] addChit():clickhouse/clickhouse-secure-svc-lb
I1027 13:19:13.928851       1 worker-chit-reconciler.go:41] addChit():clickhouse/clickhouse-secure-svc-lb:Add CHIT: clickhouse/clickhouse-secure-svc-lb
```

No reproducing steps, sorry. CHIT polcy was manual at first then I change it to "auto", maybe it helps.

Operator version: 0.21.3

**Comment by baznikin** [comment]
Here I delete manual CHIT `clickhouse-secure-svc-lb` and create `clickhouse-secure-svc` instead. After applying there is 10 templates in logs in matter of seconds...

```
I1027 13:27:02.471445       1 normalizer.go:137] applyCHITemplates():clickhouse/clickhouse:Found auto-templates num: 1
I1027 13:27:02.471481       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 13:27:02.471497       1 normalizer.go:149] applyCHITemplates():clickhouse/clickhouse:Found manual-templates num: 1
I1027 13:27:02.471522       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 13:27:02.471558       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: /clickhouse-secure-svc-lb. Selector: map[] matches labels: map[]
I1027 13:27:02.471582       1 normalizer.go:202] applyCHITemplates():clickhouse/clickhouse:Used templates count: 2
I1027 13:27:02.472623       1 worker.go:617] clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Update users IPS-2
I1027 13:27:02.690018       1 worker.go:638] finalizeReconcileAndMarkCompleted():clickhouse/clickhouse/bdf55d50-2e83-487c-93e9-db8277778b32:reconcile completed successfully, task id: bdf55d50-2e83-487c-93e9-db8277778b32
I1027 13:27:03.089138       1 worker-chi-reconciler.go:111] worker-chi-reconciler.go:52:reconcileCHI():end:clickhouse/clickhouse
I1027 13:27:03.089170       1 worker.go:416] worker.go:382:updateCHI():end:clickhouse/clickhouse
I1027 14:10:56.444816       1 worker-chit-reconciler.go:57] updateChit():clickhouse/clickhouse-secure-svc-lb:ResourceVersion change: 73850977 to 74352581
I1027 14:10:56.444856       1 worker-chit-reconciler.go:59] updateChit():clickhouse/clickhouse-secure-svc-lb:Update CHIT: clickhouse/clickhouse-secure-svc-lb
I1027 14:13:09.773853       1 worker-chit-reconciler.go:57] updateChit():clickhouse/clickhouse-secure-ssl:ResourceVersion change: 74328837 to 74353447
I1027 14:13:09.773946       1 worker-chit-reconciler.go:59] updateChit():clickhouse/clickhouse-secure-ssl:Update CHIT: clickhouse/clickhouse-secure-ssl
I1027 14:13:29.088122       1 worker-chit-reconciler.go:57] updateChit():clickhouse/clickhouse-secure-ssl:ResourceVersion change: 74353447 to 74353575
I1027 14:13:29.088166       1 worker-chit-reconciler.go:59] updateChit():clickhouse/clickhouse-secure-ssl:Update CHIT: clickhouse/clickhouse-secure-ssl




I1027 14:17:43.319990       1 worker-chit-reconciler.go:69] deleteChit():clickhouse/clickhouse-secure-svc-lb
I1027 14:17:43.320024       1 worker-chit-reconciler.go:72] deleteChit():clickhouse/clickhouse-secure-svc-lb:Delete CHIT: clickhouse/clickhouse-secure-svc-lb
I1027 14:17:43.536237       1 worker-chit-reconciler.go:38] addChit():clickhouse/clickhouse-secure-svc
I1027 14:17:43.536271       1 worker-chit-reconciler.go:41] addChit():clickhouse/clickhouse-secure-svc:Add CHIT: clickhouse/clickhouse-secure-svc
I1027 14:17:44.113436       1 controller.go:566] ENQUEUE new ReconcileCHI cmd=update for clickhouse/clickhouse
I1027 14:17:44.113655       1 worker.go:381] worker.go:381:updateCHI():start:clickhouse/clickhouse
I1027 14:17:44.113806       1 worker.go:423] Operator IPs. Previous: 10.244.0.132 Cur: 10.244.0.132
I1027 14:17:44.113826       1 worker.go:432] Operator IPs are the same. It is restart on the same IP
I1027 14:17:44.114060       1 worker.go:440] Operator is not just started. May not be clean restart
I1027 14:17:44.116638       1 worker-chi-reconciler.go:51] worker-chi-reconciler.go:51:reconcileCHI():start:clickhouse/clickhouse
I1027 14:17:44.116672       1 worker-chi-reconciler.go:55] reconcileCHI():clickhouse/clickhouse:has ancestor, use it as a base for reconcile
I1027 14:17:44.116730       1 normalizer.go:137] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Found auto-templates num: 9
I1027 14:17:44.116745       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.116757       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.116785       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.116815       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.116832       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.116874       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.116900       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.116913       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.116949       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.116969       1 normalizer.go:149] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Found manual-templates num: 1
I1027 14:17:44.116990       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.117043       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.117070       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.117109       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.117135       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.117154       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.117202       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.117228       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.117249       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
W1027 14:17:44.117282       1 normalizer.go:165] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Skip template: /clickhouse-secure-svc-lb UNABLE to find listed template. 
I1027 14:17:44.117312       1 normalizer.go:202] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Used templates count: 9
I1027 14:17:44.125086       1 worker.go:291] clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:IPs of the CHI clickhouse/clickhouse: [10.244.1.126]
I1027 14:17:44.125126       1 normalizer.go:137] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Found auto-templates num: 9
I1027 14:17:44.125139       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.125149       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.125159       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.125172       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.125184       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.125198       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.125210       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.125222       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.125233       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.125245       1 normalizer.go:149] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Found manual-templates num: 1
I1027 14:17:44.125264       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.126334       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.126372       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.126393       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.126415       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.126435       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.126720       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.126762       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.126808       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
W1027 14:17:44.126840       1 normalizer.go:165] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Skip template: /clickhouse-secure-svc-lb UNABLE to find listed template. 
I1027 14:17:44.126861       1 normalizer.go:202] applyCHITemplates():clickhouse/clickhouse/6b7614a0-cb63-4c03-843d-be02ba9da986:Used templates count: 9
I1027 14:17:44.131618       1 normalizer.go:137] applyCHITemplates():clickhouse/clickhouse:Found auto-templates num: 9
I1027 14:17:44.131647       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.131658       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.131668       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.131677       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.131686       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.131697       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.131709       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.131719       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.131730       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.131744       1 normalizer.go:149] applyCHITemplates():clickhouse/clickhouse:Found manual-templates num: 1
I1027 14:17:44.131761       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.131788       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.131808       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.132308       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.132333       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.132353       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.132369       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.132387       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.132405       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.132428       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: /clickhouse-secure-svc. Selector: map[] matches labels: map[]
I1027 14:17:44.132452       1 normalizer.go:202] applyCHITemplates():clickhouse/clickhouse:Used templates count: 10
I1027 14:17:44.142524       1 worker.go:291] clickhouse/clickhouse/dd265ad3-ffcc-4634-83fe-a0a4bdd672fb:IPs of the CHI clickhouse/clickhouse: [10.244.1.126]
I1027 14:17:44.142576       1 normalizer.go:137] applyCHITemplates():clickhouse/clickhouse:Found auto-templates num: 9
I1027 14:17:44.142771       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.142793       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.142815       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.142828       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.142849       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.142925       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.143460       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.143549       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.143598       1 normalizer.go:139] applyCHITemplates():clickhouse/clickhouse:Adding auto-template to list of possible templates: clickhouse/clickhouse-secure-ssl 
I1027 14:17:44.143643       1 normalizer.go:149] applyCHITemplates():clickhouse/clickhouse:Found manual-templates num: 1
I1027 14:17:44.144439       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.144792       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.144842       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.144876       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.144919       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.145014       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.145114       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.145141       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.145201       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: clickhouse/clickhouse-secure-ssl. Selector: map[] matches labels: map[]
I1027 14:17:44.145280       1 normalizer.go:181] applyCHITemplates():clickhouse/clickhouse:Apply template: /clickhouse-secure-svc. Selector: map[] matches labels: map[]
I1027 14:17:44.145309       1 normalizer.go:202] applyCHITemplates():clickhouse/clickhouse:Used templates count: 10
```

**Comment by alex-zaitsev** [comment]
@baznikin , there is no duplication. But logs are definitely misleading here, we will fix it.

**Comment by sunsingerus** [comment]
This issues is being worked on. Plan to release in 0.23.0
Meanwhile, feel free to check dev branch. Image is available on dockerhub

**Comment by sunsingerus** [comment]
released in 0.22.1
